app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
Well, that's why I framed this as a question :wink:

The thing is, it's better to default to a uniform presentation, and that's what CSS is good at. Better have a specific opt-out class for the specific cases where you don't want to apply the style, than to repeat a specific class name in 90% of the cases. And I personally find it more expressive (clearer intention) to read “all labels that follow a radio button or a checkbox look like a button” than to read “the elements that have the `inputButton` class look like buttons”.

Now of course, it depends on how widely this behaviour is the expected one.